include/z64math.h

```diff
@@ -1,7 +1,7 @@
 #ifndef Z64MATH_H
 #define Z64MATH_H
 
-#include "ultra64.h"
+#include "PR/ultratypes.h"
```

Comment:
By changing this in `z64math.h`, wouldn't this expose `sys_math3d.c` to more files than necessary? Or was `sys_math3d.c` already exposed to this in a sneaky way?

Reply:
Actually ignore, I need a coffee. It's the other way around, it's exposing less of it.